map/framework.cpp

```diff
@@ -1164,25 +1100,48 @@ bool Framework::GetGroupCountryIdFromFeature(FeatureType const & ft, string & na
   return false;
 }
 
+bool Framework::SearchEverywhere(search::SearchParams params)
```

Comment:
In any case, you're right - need to create EverywhereSearchParams for SearchEverywhere. Passing raw SearchParams is bad, because it's quite general.